src/main/java/com/actions/emislabbackend/model/OperationTeeth.java

```diff
@@ -0,0 +1,106 @@
+package com.actions.emislabbackend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.text.DecimalFormat;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OperationTeeth {
+
+    private int id;
+    private String medic;
+    private String patient;
+    private WorkType worktype;
+    private int teethNr;
+
+
+    public OperationTeeth(int id, String medic, String patient, WorkType worktype, int teethNr) {
+        this.id = id;
+        this.medic = medic;
+        this.patient = patient;
+        this.worktype = worktype;
+        this.teethNr = teethNr;
+        calculateProcent(this.worktype, this.teethNr);
+    }
+
+    private double timeea;
+    private double paula;
+    private double emi;
+
+
+    String pattern = "##.##";
+    DecimalFormat decimalFormat = new DecimalFormat(pattern);
+    String format = decimalFormat.format(123456789.123);
+
+
+    public void calculateProcent(WorkType wt, int teethNr) {
+
+        switch (wt) {
+            case METALO_CERAMICA:
+                setTimeea(0.41 * wt.getPrice() * teethNr);
+                setPaula(0.105 * wt.getPrice() * teethNr);
+                setEmi(0.485 * wt.getPrice() * teethNr);
+                break;
+            case METALO_CERAMICA_ONE:
+                setTimeea(0.335 * wt.getPrice() * teethNr);
+                setPaula(0.105 * wt.getPrice() * teethNr);
+                setEmi(0.56 * wt.getPrice() * teethNr);
+                break;
+            case MCP_FIZIONOMIC:
+                setTimeea(0.335 * wt.getPrice() * teethNr);
+                setPaula(0.105 * wt.getPrice() * teethNr);
+                setEmi(0.56 * wt.getPrice() * teethNr);
+                break;
+            case CERAMICA_ZIRKONIU:
```

Comment:
Switch case is rarely a good practice. Try using strategy pattern instead, also why do you keep the income in variables. What happens if I hire a new technician?